### DISCUSSION TITLE
Is there a dictionary for en-CA?

### DISCUSSION THREAD

**Discussion by talyh** [streetsidesoftware/cspell-dicts#1952] [Q&A] [answered]
I tried setting `en-CA` as the language for my spell checker, but I'm still getting errors for Canadian spelling

<img width="456" alt="image" src="https://user-images.githubusercontent.com/23743595/221045451-7433538b-6232-4447-a8f0-d088adc86922.png">

<img width="202" alt="image" src="https://user-images.githubusercontent.com/23743595/221045428-a6e17156-4c09-4032-9959-f2910e193086.png">

What's the right way to refer to the Canadian dictionary if one exists?
And what's the right way to add one if one doesn't exist?

**Comment by Jason3S** [comment, marked as answer]
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker-canadian-english
